context.go

```diff
@@ -68,6 +71,47 @@ type Helper interface {
 	TrackProgress(percent int, desc string, reserveUntil *time.Time) error
 }
 
+// The test helper is a mode of operation which tries to avoid network calls.
+// This allows you to unit test your job code with the Helper API without
+// Faktory running.
+type testHelper struct {
+	job        *faktory.Job
+	pushedJobs []*faktory.Job
+}
+
+func (h *testHelper) Jid() string {
+	return h.job.Jid
+}
+func (h *testHelper) Bid() string {
+	if b, ok := h.job.GetCustom("_bid"); ok {
+		return b.(string)
+	}
+	return ""
+}
+func (h *testHelper) JobType() string {
+	return h.job.Type
+}
+
+func (h *testHelper) TrackProgress(percent int, desc string, reserveUntil *time.Time) error {
+	return nil
```

Comment:
Instead of creating a separate type which doesn't implement most of the interface, maybe use a swappable Push() fn?